content/docs/command-reference/dag.md

```diff
@@ -15,25 +15,8 @@ positional arguments:
 
 ## Description
 
-A data pipeline, in general, is a series of data processing
-[stages](/doc/command-reference/run) (for example, console commands that take an
-input and produce an <abbr>output</abbr>). A pipeline may produce intermediate
-data, and has a final result.
-
-Data science and machine learning pipelines typically start with large raw
-datasets, include intermediate featurization and training stages, and produce a
-final model, as well as accuracy [metrics](/doc/command-reference/metrics).
-
-In DVC, pipeline stages and commands, their data I/O, interdependencies, and
-results (intermediate or final) are specified in `dvc.yaml`, which can be
-written manually or built using the helper command `dvc run`. This allows DVC to
-restore one or more pipelines later (see `dvc repro`).
-
-> DVC builds a dependency graph
-> ([DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) to do this.
-
-`dvc dag` command displays the stages of a pipeline up to the target stage. If
-`target` is omitted, it will show the full project DAG.
+The `dvc dag` command displays the stages of a <abbr>data pipeline</abbr> up to
+the target stage. If `target` is omitted, it will show the full project DAG.
```

Comment:
We tend to go straight to the verb in these intros:
```suggestion
Displays the stages of a <abbr>data pipeline</abbr> up to the target stage. If
`target` is omitted, it will show the full project DAG.
```
(committing)